### PR TITLE
Automatically flag apps as deprecated-software with the help of a crappy script

### PR DESCRIPTION
**Pull request by Salamandar**
tl;dr, only working for github upstreams, checks if the latest commit is more than 1 year old

### CODE REVIEW

**Comment by zamentur** [comment]
For me we can remove from this list a lot of html/js apps (without server side code). Specifically all games.

So at least: 20euros, 243-game, hextris, anarchism, cubiks-2048, encryptor-decryptor, filepizza, hat, osjs, simple-hash-generator, strut, trustyhash, tyto

Those apps are not really deprecated, those apps are finished and doesn't really need update for now.

**Comment by orhtej2** [comment]
Adguard home is up to date and released 2 days ago 🤔

**Comment by Salamandar** [comment]
> Adguard home is up to date and released 2 days ago 🤔

hmmm indeed. i'm going to need to test this more deeply.

File: .github/workflows/deprecated.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+      - name: Install toml python lib
+        run: |
+          pip3 install toml tomlkit gitpython
+
```
Comment (tituspijean) [inline .github/workflows/deprecated.yml:20]: From contributors meeting: we need to actually run the script somewhere around here :p

**Pull request closed without merging**